### DISCUSSION TITLE
%5B and %5D in URLs instead of [ and ]

### DISCUSSION THREAD

**Discussion by drnkwtr** [spatie/laravel-query-builder#948] [Q&A]
Hello, dear community and developers!

Today i met a problem on submitting form. Each filter[any] gone to filter%5Bany%5D. It looks pretty bad as i think, so iirc it can't be fixed in URL, so is there any way to replace ?filter[any] to ?any=123&any_other=1234?

Thanks a lot for your replies, so appreciate.

**Comment by peterjthomson** [comment]
The unicode brackets are more strictly 'web compliant' so some of the Laravel url helpers can cast things to unicode whether we like it not. This is also an issue on the 'reading' side of things with the request helpers. One approach is to build everything to work with both or to aggressively cast to either [ ] brackets or unicode everywhere. I'm hoping that the recent Laravel URL and Request helper upgrades might nudge the community towards a common standard.
